### ISSUE TITLE
Invalid usage of publishCreate() :: Values must have an `undefined`, instead got ::

### ISSUE THREAD

**Issue by dcbartlett** [balderdashy/sails#2105]
Sails: v0.10.3
sails-mysql: ^0.10.5
I've experiencing the error in the title when using the default blueprint controller for create.

Here is the error and Model.

```
error: Invalid usage of publishCreate() :: Values must have an `undefined`, instead got ::
{ name: 'bob',
  state: 1,
  id: 1,
  createdAt: Tue Aug 12 2014 02:51:46 GMT-0500 (CDT),
  updatedAt: Tue Aug 12 2014 02:51:46 GMT-0500 (CDT) }
```

```javascript
/**
* Class.js
*
* @description :: TODO: You might write a short summary of how this model works and what it represents here.
* @docs        :: http://sailsjs.org/#!documentation/models
*/

module.exports = {
    attributes: {
        id: {
            type: 'integer',
            unique: true
        },
        name: {
            type: 'string'
        },
        state: {
            type: 'integer'
        }
    }
};
```

**Comment by Dallas62** [comment]
Is that solved?

**Comment by svenseitan** [comment]
It's ok for me, thanks !

**Comment by CWyrtzen** [comment]
Is this resolved? Does the latest release clear things up? Can we close this?

**Comment by sailsbot** [comment]
Thanks for posting, @dcbartlett.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!

**Comment by jlno** [comment]
I removed `toJSON` function from my model and work.
